{{ cookiecutter.project_shortname }}/run-tests.sh

```diff
@@ -4,8 +4,6 @@
 # TODO: Pass the services required by your module to the
 # docker-services-cli e.g. `docker-services-cli up es postgresql redis`
 docker-services-cli up
-python -m pydocstyle {{ cookiecutter.package_name }} tests docs && \
```

Comment:
@lnielsen @ppanero why not adding:
`docker-services-cli up && \`

@topless had an issue with the docker-compose not running, tests were running, but it was not straightforward to spot that issue...
If docker-service-cli fails, we should fail and stop.
WDYT?